XamlControlsGallery/ControlPages/SplitButtonPage.xaml

```diff
@@ -22,7 +22,7 @@
                 </Grid.ColumnDefinitions>
 
                 <muxc:SplitButton x:Name="myColorButton" AutomationProperties.Name="Font color" Padding="0" MinHeight="0" MinWidth="0" VerticalAlignment="Top" Click="myColorButton_Click">
-                    <Rectangle x:Name="CurrentColor" Width="{StaticResource SwatchSize}" Height="{StaticResource SwatchSize}" Fill="Black" Margin="0"/>
+                    <Rectangle x:Name="CurrentColor" Width="{StaticResource SwatchSize}" Height="{StaticResource SwatchSize}" Fill="Gray" Margin="0"/>
```

Comment:
Instead of Gray, could we make the default some obviously obnoxious color like Teal? I'm worried that Gray will make the control look like it's in the Disabled state, but it's not. If we choose an obviously customized color, it'll help indicate that a non-default color has been set.

Reply:
Yeah, gray wasn't a great choice. I've switched to green now as it is one of the colors we had in the selection, that also is fairly  visible in both themes, and isn't to aggressive to the eye.